lib/rails_container/.ruby-version

```diff
@@ -0,0 +1 @@
+2.3.3
```

Comment:
This is inconsistent with the ruby version used in the Dockerfile. It would be better to use the same version (preferably the newest possible) to make sure the ruby code we write behaves the same on our local and prod.

Reply:
sorted - thenks